fix build (#59)
davidben/tls-trust-expressions: draft-beck-tls-trust-anchor-ids.md

When the extension is sent in Certificate, the `extension_data` MUST be empty and indicates that the sender sent the certificate because the certificate matched a trust anchor identifier sent by the peer. When used in this form, the extension may only be sent in the first CertificateEntry. It MUST NOT be sent in subsequent ones.

> ## Certificate Selection {#certificate-selection}

Reviewer: Nit: You only need this if you're changing from the default label. (Default label is lowercase with hyphens.)

Reviewer: ```suggestion
## Certificate Selection
```

{{?I-D.ietf-tls-cert-abridge}} proposes to predistribute known intermediate certificates to relying parties, as a compression scheme. A multi-certificate deployment model provides another way to achieve this effect. To relying parties, a predistributed intermediate certificate is functionally equivalent to a root certificate. PKIs use intermediate certificates because changing root certificates requires updating relying parties, but predistributed intermediates already presume updated relying parties.

> A CA operator could provide subscribers with two certification paths: a longer path ending at a long-lived trust anchor and shorter path the other ending at a short-lived, revocable root. Relying parties would be configured to trust both the long-lived root and the most recent short-lived root. The negotiation mechanism in {{certificate-selection}} would then send the shorter path to up-to-date relying parties, and the longer path to older relying parties.

Reviewer: Super nitpicky nitpick: it's really the whole protocol, not just the selection algorithm, since sometimes we effectively invert selection. TBH since it's clearly in the context of this (much shorter) document, I think we can just write...

```suggestion
A CA operator could provide subscribers with two certification paths: a longer path ending at a long-lived trust anchor and shorter path the other ending at a short-lived, revocable root. Relying parties would be configured to trust both the long-lived root and the most recent short-lived root. A server that prioritizes the shorter path would then send the shorter path to up-to-date relying parties and the longer path to older relying parties.
```
